### DISCUSSION TITLE
Typescript definitions missing for v7.6.0

### DISCUSSION THREAD

**Discussion by GourmetGit** [TanStack/table#2773] [General]
Currently there are no typescript definitions for version v7.6.0 to be able to use the new useGridLayout. If someone could submit this to https://www.npmjs.com/package/@types/react-table I would be really grateful. 

Or do you have any guide how to override the missing typescript definitions locally?

**Comment by macrozone** [comment]
we wanted to use this library for a bigger project, but i was quite shocked that it has no proper typings.

would be really glad as well to see an update for the current version. looking for alternatives as well.

**Reply by tannerlinsley** [reply]
Currently, the types for v7 are maintained by the community in Definitely Typed. 

v8 (currently under construction) will be written natively in Typescript

**Reply by billsaysthis** [reply]
what's the state of v8, @tannerlinsley ? We're just starting with React-Table but I'm having a terrible time at runtime even though I installed the `@types/react-table` and also added the `react-table-config.ts` as suggested. (https://github.com/ggascoigne/react-table-example/issues/1)

Having the same issue trying to use GlobalFilter.
